dokku

```diff
@@ -101,13 +101,13 @@ case "$1" in
           START_CMD=$(dokku config:get $APP DOKKU_DOCKERFILE_START_CMD || $START_CMD)
         fi
 
+        [[ "$PROC_TYPE" == "web" ]] && port=${DOKKU_DOCKERFILE_PORT:=5000}
```

Comment:
Like, we check several times what the PROC_TYPE is, as well as two checks on BIND_EXTERNAL.